examples/validate_CT_errors.py

```diff
@@ -0,0 +1,165 @@
+""" Validate Inactive CT Error
+==============================
+
+Estimates the error of the integral of an actual data set of a standard echo
+experiment. Three methods of acquiring the error associated with the data are 
+compared:
```

Comment:
when we meet, I'll also diff this against the real data one to make sure we are using the exact same code.